README.md

```diff
@@ -232,17 +232,29 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 
 ### Build Cloud
 
-```shell
-cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
-make # or `make edgecontroller`
-```
++ Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller/conf/ontroller.yaml` configuration file to modify `cloudhub.ca`, `cloudhub.cert`, `cloudhub.key` to generated Certificate path
+
++ Build Cloud
+    ```shell
+    cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
+    make # or `make edgecontroller`
+    ```
 
 ### Build Edge
 
-```shell
-cd $GOPATH/src/github.com/kubeedge/kubeedge/edge
-make # or `make edge_core`
-```
++ Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/edge/conf/edge.yaml` configuration file
+  + Replace `edgehub.websocket.certfile` and `edgehub.websocket.keyfile` with your own certification path
+  + In websocket:url you need to give IP address of the master.
+  + replace `fb4ebb70-2783-42b8-b3ef-63e2fd6d242e` with edge node ip in edge.yaml in 3 place.
```

Comment:
Should we have this step in README to replace the id with IP ? It looks like it is a compulsion to replace with IP. can we mention something like metadata.name in node.json and the the id in these 3 places should be same.
Thoughts ? @edisonxiang @rohitsardesai83

Reply:
+1:)

Reply:
I didn't initially build a kubeedge cluster successfully because of the wrong edge.yaml, see [kubeedge/kubeedge#258](https://github.com/kubeedge/kubeedge/issues/258#issuecomment-474224279) .So I think it should be added to REAMDE to prevent others from experiencing similar problems. Maybe we can provide a more detailed explanation?

Reply:
yep. we can add the comments here.
Did you use the default `node.json` to create the node?
https://github.com/kubeedge/kubeedge/blob/master/build/node.json#L5